dev/io.openliberty.jakarta.xmlBinding.3.0/bnd.bnd

```diff
@@ -31,16 +31,17 @@ DynamicImport-Package: \
 #Bundle-Activator: org.apache.geronimo.osgi.locator.Activator
 
 Include-Resource:\
-  @${repo;jakarta.xml.bind:jakarta.xml.bind-api;3.0.0;EXACT}!/!META-INF/maven/*, \
-  jakarta/xml/bind=${bin}/jakarta/xml/bind
+  @${repo;jakarta.xml.bind:jakarta.xml.bind-api;3.0.1;EXACT}!/!(META-INF/maven/*|module-info.class)
 # @${repo;org.apache.geronimo.specs:geronimo-jaxb_2.2_spec;1.0.1}!/org/apache/geronimo/osgi/locator/*, \
```

Comment:
Do you know what these commented out lines are for?

Reply:
The bnd originally was copied from the 2.2 version and these were determined not to be needed I assume.  @neuwerk would know for sure.

Reply:
Yeah, they were just stale code from 2.2, I should have deleted them at the time.

Reply:
I removed them.